lib/room.cpp

```diff
@@ -684,6 +684,11 @@ void Room::addTag(const QString& name, const TagRecord& record)
     d->broadcastTagUpdates();
 }
 
+void Room::addTag(const QString &name, int order)
```

Comment:
Sorry but, maybe a bit counter-intuitively, `order` is `QString`. See `events/accountdataevents.h` for the full definition of `TagRecord`.